### ISSUE TITLE
Node JS: crypto import polyfill not taken into account?

### ISSUE THREAD

**Issue by matteocargnelutti** [webrecorder/warcio.js#49]
Hi there 👋 !

I see that there is now a [polyfill to automatically import and declare `crypto` in the global scope](https://github.com/webrecorder/warcio.js/blob/main/src/polyfills.cjs#L10).

Unfortunately, I'm getting the following error when using `WARCRecord.create()` after removing my own global import of `crypto` : 
```bash
ReferenceError: crypto is not defined
```

For reference, I'm importing `crypto` as such to compensate for its absence:
```javascript
import crypto from 'crypto'
global.crypto = crypto
```

I'm using `warcio.js 2.0.1` with Node 18+.

Thanks in advance,

**Comment by ikreymer** [comment]
Yes, the current polyfill applies only to the test suite. I was having trouble setting up a proper polyfill for the node build.
Though, Node 18 does have `--experimental-global-webcrypto` for making crypto global and I guess in node 19, this is the default..

**Comment by jlarmstrongiv** [comment]
Finally found a way to have automatic polyfills:

Use [`inject`](https://github.com/mswjs/msw/blob/main/tsup.config.ts#L54) with a separate [polyfills file](https://github.com/mswjs/msw/blob/main/config/polyfills-node.ts)
